src/Screens/PokemonIndex.tsx

```diff
@@ -1,28 +1,46 @@
 import usePokemons from "@/hooks/usePokemons";
 import React from "react";
-import { View, StyleSheet, FlatList } from "react-native";
-import PokemonItem from "../components/PokemonItem";
+import { View, StyleSheet, FlatList, Image, SafeAreaView } from "react-native";
+import PokemonItem from "@components/PokemonItem";
 
 const PokemonIndex = () => {
   const { pokemonIndex } = usePokemons();
 
   return (
-    <View style={styles.container}>
+    <SafeAreaView style={styles.container}>
+      <View style={styles.logoContainer}>
+        <Image
+          style={styles.logoImage}
+          resizeMode="contain"
+          source={require("assets/images/pokemon_logo.png")}
+        />
+      </View>
+
       <FlatList
-        contentContainerStyle={{ gap: 8, marginHorizontal: 16 }}
-        columnWrapperStyle={{ gap: 8 }}
-        numColumns={2}
+        contentContainerStyle={{ gap: 16, marginHorizontal: 16 }}
         data={pokemonIndex}
         renderItem={({ item }) => (
-          <PokemonItem shortName={item.name} imageUri={item.imageUri} />
+          <PokemonItem
+            shortName={item.name}
+            imageUri={item.imageUri}
+            types={item.types}
+          />
         )}
         keyExtractor={(item) => item.name}
       />
-    </View>
+    </SafeAreaView>
   );
 };
 
 const styles = StyleSheet.create({
+  logoImage: { flex: 1, height: "100%" },
+  logoContainer: {
+    height: 100,
```

Comment:
FYI: w RN jeśli ładujesz obrazki z sieci, trzeba (chyba że naprawili, ale tak czy siak -warto to zrobić) podać jego rozmiar. Tak żeby renderer miał wiedzę ile miejsca na ekranie zajmie obrazek już w czasie rysownia ekranu